views/application.py

```diff
@@ -38,3 +43,51 @@ def get_queryset(self):
         """
 
         return Application.objects.filter(team_members=self.request.person)
+
+    def update(self, request, *args, **kwargs):
+        """
+        update membership status of the application member
+        :param request: the request being processed
+        :param args: arguments
+        :param kwargs: keyword arguments
+        :return: updated instance
```

Comment:
This does not return the updated instance. It returns a response for the request.